.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# Pull Request Template
+
+## Description
+
+Fix #(issue)
+
+<!-- Please include a summary of the change and which issue is fixed.
+Please also include relevant motivation and context.
+List any dependencies that are required for this change. -->
+
+
+## Type of change
+
+Please check the option that is related to your PR.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
+- [ ] Documentations (modification for documents)
+
+## Checklist
+
+- [ ] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
+  - [ ] Run `yapf -i <modified file>` and check no errors
```

Comment:
One small comment here - should we specify the required version of yapf (0.22.0) in this template and some other places? I know this trips up new contributors occasionally.

Reply:
+1!

Reply:
I fixed!